Content.Server/Instruments/SwappableInstrumentComponent.cs

```diff
@@ -11,4 +11,11 @@ public sealed partial class SwappableInstrumentComponent : Component
     /// </summary>
     [DataField("instrumentList", required: true)]
     public Dictionary<string, (byte, byte)> InstrumentList = new();
+
+    // Frontier: harpy instruments
+    /// <summary>
+    /// When true, only the instrument entity itself can swap its sound.
+    /// </summary>
+    [DataField]
+    public bool OnlySetBySelf = true;
```

Comment:
This is true by default, meaning that ALL swappable instruments are affected by this change. You didn't modify the yaml prototypes of other instruments, meaning that all swappable instruments (synthesizers, guitars, etc) are affected by this change and cannot be swapped by anyone unless they get possessed by an admin...

You should make it false by default, and set it to true on MapInit in SingerSystem instead.

Reply:
okie dokie